test/fakefs_test.rb

```diff
@@ -1292,6 +1292,7 @@ def test_should_set_pos_with_tell_method
   ]
 
   (RealFile.instance_methods - OMITTED_FILE_METHODS).each do |method_name|
+    next if defined?(JRUBY_VERSION) && [:ttymode, :ttymode_yield, :to_channel, :to_outputstream, :to_inputstream].include?(method_name)
```

Comment:
@bquorning Thanks for reading! I was loath to create new local variables, though. If that's acceptable, that's grand!